src/main/pg/gimbal.c

```diff
@@ -0,0 +1,46 @@
+/*
+ * This file is part of Betaflight.
+ *
+ * Betaflight is free software. You can redistribute this software
+ * and/or modify this software under the terms of the GNU General
+ * Public License as published by the Free Software Foundation,
+ * either version 3 of the License, or (at your option) any later
+ * version.
+ *
+ * Betaflight is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ *
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this software.
+ *
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "platform.h"
+
+#ifdef USE_GIMBAL
+
+#include "pg/pg.h"
+#include "pg/pg_ids.h"
+
+#include "gimbal.h"
+
+PG_REGISTER_WITH_RESET_TEMPLATE(gimbalTrackConfig_t, gimbalTrackConfig, PG_GIMBAL_TRACK_CONFIG, 0);
+
+PG_RESET_TEMPLATE(gimbalTrackConfig_t, gimbalTrackConfig,
+    // Default to full gain and no offset
+    .gimbal_roll_gain   = 40,
+    .gimbal_roll_offset = 0,
+    .gimbal_pitch_thr_gain  = 10,
+    .gimbal_pitch_low_gain  = 20,
+    .gimbal_pitch_high_gain = -10,
+    .gimbal_pitch_offset = 10,
+    .gimbal_yaw_gain    = 20,
+    .gimbal_yaw_offset  = 0,
+    .gimbal_stabilisation = 0,
+    .gimbal_sensitivity = 15
```

Comment:
(defaulting to maximum value ? Also, what is negative sensitivity good for?)

Reply:
It's the range you'll find mentioned in https://cdn.shopify.com/s/files/1/0036/3921/4169/files/GM_Series_Manual_V1.0_1.pdf as **Follow Sensitivity**. I've no idea why they chose to do it that way.

Reply:
As for the default values, these are my stab as something usable. These will most likely get revised to something more flyable once we get some feedback.